src/test/java/dev/fastgql/TestUtils.java

```diff
@@ -26,4 +30,26 @@ public static String readResource(String name, VertxTestContext context) {
     }
     return resource;
   }
+
+  public static Stream<String> queryDirectories() throws IOException {
+    int resourceRootNameCount = getResourceRoot().getNameCount();
+    Path basePath = getBasePath("queries");
+    Stream<Path> stream = Files.walk(basePath, 2);
+    return stream
+        .filter(
+            path ->
+                Files.isDirectory(path)
+                    && !path.getParent().equals(basePath)
+                    && !path.equals(basePath))
+        .map(path -> path.subpath(resourceRootNameCount - 1, path.getNameCount()))
```

Comment:
we can just define final variable which is getNameCount()-1 and avoid doing some calculations here